TestProject/gameoflife.php

```diff
@@ -2,36 +2,59 @@
 use cellularAutomat\Board;
 use options\Getopt;
 
+use input\Base;
+use input\Glider;
+use input\Random;
+
+$width = 10;
+$height = 10;
+$maxSteps = 20;
 
 /**
  * Function for Autoloader.
  * @param $className
  */
 function myAutoload($className) {
-    $classPath = str_replace("\\", "/", $className);
-    require_once(sprintf("%s/%s.php", __DIR__, $classPath));
+    $classPath = sprintf("%s/%s.php", __DIR__, str_replace("\\", "/", $className));
+    if (is_readable($classPath)) require_once($classPath);
 }
 
 /**
- * Execution of the autoloader
+ * Function for the autoloader.
```

Comment:
A better comment would be: "Register our autoloader." But this can also be done with a small single-line comment like `//...`.